play/src/front/Phaser/Game/CameraManager.ts

```diff
@@ -361,4 +363,45 @@ export class CameraManager extends Phaser.Events.EventEmitter {
             zoom: this.camera.scaleManager.zoom,
         };
     }
+
+    private explorationMouseIsActive = false;
+    // Create function to define the camera on exploration mode. The camera can be moved anywhere on the map. The camera is not locked on the player. The camera can be zoomed in and out. The camera can be moved with the mouse. The camera can be moved with the keyboard. The camera can be moved with the touchpad.
+    public setExplorationMode(): void {
+        this.cameraLocked = false;
+        this.stopFollow();
+        this.cameraLocked = false;
+        this.setCameraMode(CameraMode.Exploration);
+        this.scene.cameras.main.setBounds(
+            -this.cameraBounds.x,
+            -this.cameraBounds.y,
+            this.cameraBounds.x * 3,
+            this.cameraBounds.y * 3,
+            true
+        );
```

Comment:
I love this!
Don't you think we should also use it in main mode too?
This could help us to center the map when the map is really small (rather than having the map stuck in upper left)

Reply:
Yes I think so and it will be nice 😍